### PR TITLE
Upgrade tsd to typings, Related to #9658

### PR DESCRIPTION
**Pull request by sudheesh001**

### CODE REVIEW

**Comment by weswigham** [comment]
So.... We don't actually use this task anymore. We even cut it from the gulpfile. We were planning on just straight up removing the tsd dev dependency, since we can now use @types for what we need.

**Comment by sudheesh001** [comment]
@weswigham I've updated the pull request to remove the tsd dependency, Please let me know in case there are more changes needed :smile:

File: Jakefile.js [hunk 1]
```diff
@@ -466,14 +466,8 @@ task("publish-nightly", ["configure-nightly", "LKG", "clean", "setDebugMode", "r
...
 
-var scriptsTsdJson = path.join(scriptsDirectory, "tsd.json");
-file(scriptsTsdJson);
-
-task("tsd-scripts", [scriptsTsdJson], function () {
-    var cmd = "tsd --config " + scriptsTsdJson + " install";
-    console.log(cmd);
-    exec(cmd);
-}, { async: true });
+var scriptsTypingsJson = path.join(scriptsDirectory, "typings.json");
```
Comment (mhegazy) [inline Jakefile.js]: do not think we need this either.

**Comment by sudheesh001** [comment]
@weswigham Updated the pull request :smile:

**Comment by weswigham** [comment]
Seems good. I don't see any other vestiges of tsd left in the repo, so I'll merge it.

**Pull request merged**